docs/metrics.md

```diff
@@ -1,37 +1,140 @@
+runtime metrics.
 
 ## Configuration
 
 For configuration options, see [advanced configuration](advanced-config.md#metrics).
 
-## Runtime metrics
+## Usage (custom metrics)
 
-The following is a list of metrics automatically collected and exported:
+Add `@opentelemetry/api-metrics` to your dependencies.
+
+```javascript
+const { startMetrics } = require('@splunk/otel');
+const { Resource } = require('@opentelemetry/resources');
+const { metrics } = require('@opentelemetry/api-metrics');
+
+// All fields are optional.
+startMetrics({
+  // Takes preference over OTEL_SERVICE_NAME environment variable
+  serviceName: 'my-service',
+  // The suggested resource is filled in via OTEL_RESOURCE_ATTRIBUTES
+  resourceFactory: (suggestedResource: Resource) => {
+    return suggestedResource.merge(new Resource({
+      'my.property': 'xyz',
+      'build': 42,
+    }));
+  },
+  exportIntervalMillis: 1000, // default: 5000
+  // The default exporter used is OTLP over gRPC
+  endpoint: 'http://collector:4317',
+});
+
+const meter = metrics.getMeter('my-meter');
+const counter = meter.createCounter('clicks');
+counter.add(3);
+```
+
+## Using custom metric readers and exporters
 
-- `nodejs.memory.heap.total` (gauge, bytes) - Heap total via `process.memoryUsage().heapTotal`.
-- `nodejs.memory.heap.used` (gauge, bytes) - Heap used via `process.memoryUsage().heapUsed`.
-- `nodejs.memory.rss` (gauge, bytes) - Resident set size via `process.memoryUsage().rss`.
-- `nodejs.memory.gc.size` (cumulative_counter, bytes) - Total collected by the garbage collector.
-- `nodejs.memory.gc.pause` (cumulative_counter, nanoseconds) - Time spent doing GC.
-- `nodejs.memory.gc.count` (cumulative_counter, count) - Number of times GC ran.
-- `nodejs.event_loop.lag.max` (gauge, nanoseconds) - Max event loop lag within the collection interval.
-- `nodejs.event_loop.lag.min` (gauge, nanoseconds) - Min event loop lag within the collection interval.
+Custom exporters and/or readers can be provided via the `metricReaderFactory` option.
 
-## Custom metrics
+:warning: Setting `metricReaderFactory` will invalidate `exportInterval` and `endpoint` options.
 
-You can send custom metrics via the internal [SignalFx client](https://github.com/signalfx/signalfx-nodejs#reporting-data):
+```javascript
+const { startMetrics } = require('@splunk/otel');
+const { PrometheusExporter } = require('@opentelemetry/exporter-prometheus');
+const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-http');
+const { PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics-base');
+
+startMetrics({
+  serviceName: 'my-service',
+  metricReaderFactory: () => {
+    return [
+      new PrometheusExporter(),
+      new PeriodicExportingMetricReader({
+        exportIntervalMillis: 1000,
+        exporter: new OTLPMetricExporter({ url: 'http://localhost:4318' })
+      })
+    ]
+  }
+});
+```
+
+## Choosing temporality aggregation type
```

Comment:
Please explain what aggregation temporality is and when it should be modified: https://github.com/open-telemetry/opentelemetry-proto/pull/192

Reply:
Added a link to the spec and tried to explain the difference between aggregation temporality options.